public/css/account.css

```diff
@@ -1,8 +1,5 @@
-body {
-    padding: 0px;
-    margin: 0px;
-    border: 0px;
-}
+@import './root.css';
```

Comment:
these are css files so I don't think you can use the `@import` syntax.

Reply:
What do you think could be done instead because MDN has it in their docs
https://developer.mozilla.org/en-US/docs/Web/CSS/@import#importing_css_rules

Reply:
It seems to be working
<img width="323" alt="Screen Shot 2022-06-27 at 2 57 01 PM" src="https://user-images.githubusercontent.com/19805355/176024926-8ae9ff12-1f64-478b-b339-ebe4a9140b15.png">

Reply:
🙌🏽